package.json

```diff
@@ -4,14 +4,18 @@
   "private": true,
   "description": "",
   "scripts": {
+    "start": "grunt dev",
     "backend": "cross-env LOAD_TARGET_FILE=./backend/index.js nodemon --ignore backend/discard/ --watch backend/ Gruntfile.js",
     "rollup": "rollup -c",
     "rollup-dev": "rollup -c -w",
     "sass-dev": "node-sass --output-style nested --source-map true -w -r -o ./dist/assets/css ./frontend/assets/style",
     "sass-dist": "node-sass --output-style compressed -o ./dist/assets/css ./frontend/assets/style",
     "server": "serve -p 8000",
     "dev": "npm-run-all --parallel backend sass-dev rollup-dev server",
+    "cy:run": "cypress run",
+    "cy:open": "cypress open",
     "test": "xvfb-run --server-args=\"-screen 0 1024x768x24\" grunt test",
+    "test:cy": "start-server-and-test start http://localhost:8000 cy:run",
```

Comment:
It's not clear to me that we need this new dependency. If we can do without adding new dependencies, we should (we have an open issue (#372) to get rid of as many of them as possible).

For example, you could look into using the existing `npm-run-all` and `serve` dependencies that we have (as used by `npm run dev` script on line 14 and 13 here), and/or you could modify the Gruntfile.babel.js file.

Reply:
`npm-run-all` was my fist try. But it didn't work. It starts the server but it seems it doesn't give any "sign" as finished, so the next command never starts.

That's what start-server-and-test is good at, [as mentioned in cypress docs](https://docs.cypress.io/guides/guides/continuous-integration.html#Boot-your-server). It specifically waits for :8000 to be ready and then starts the tests.